pkg/web/routes.go

```diff
@@ -105,6 +105,7 @@ func (s *Server) setupRoutes() (err error) {
 	s.router.GET("/counterparty", authenticate, s.CounterpartyVasps)
 	s.router.GET("/users", authenticate, s.UsersManagement)
 	s.router.GET("/utilities/travel-address", authenticate, s.TravelAddressUtility)
+	s.router.GET("/apikeys", authenticate, s.APIKeys)
```

Comment:
```suggestion
	s.router.GET("/users", authenticate, s.UsersManagement)
	s.router.GET("/apikeys", authenticate, s.APIKeys)
	s.router.GET("/utilities/travel-address", authenticate, s.TravelAddressUtility)
```

Just for organization purposes, could we keep things in the order that the routes are defined; in this case users, apikeys, utilities?